src/wp-includes/class-wp-roles.php

```diff
@@ -248,9 +248,8 @@ public function remove_cap( $role, $cap ) {
 	public function get_role( $role ) {
 		if ( isset( $this->role_objects[ $role ] ) ) {
 			return $this->role_objects[ $role ];
-		} else {
-			return null;
 		}
```

Comment:
See above code change then remove these line of code.

Reply:
@mukeshpanchal27 Done! Thank you for the suggestions